pipelines/src/pipelines/prediction.py

```diff
@@ -142,7 +133,7 @@ def xgboost_pipeline(
 
 if __name__ == "__main__":
```

Comment:
either here, or as another PR, let's take out the compilation logic from `pipeline.py` and use the KFP CLI

Reply:
agreed, let's do this in a separate PR.